.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 .vscode-test/
 *.vsix
 .DS_Store
-src/test/workspace/.vscode
+test/workspace/.vscode
+dist
```

Comment:
nit: PR tool detected there's no new line at the end of this file. 

I don't know that this will cause any issues in this case but I have seen some tools not parse the files correctly because of it before.